src/index.ts

```diff
@@ -1765,8 +1767,4 @@ declare global {
   }
 }
 
-export import JSX = JSXTypes;
-
-export namespace createElement {
```

Comment:
Trying to reexport the `JSXTypes` namespace from the `createElement` namespace fails with `Export declarations are not permitted in a namespace.ts(1194)`.

Building my JSX-based project succeeds without the `createElement` namespace and tooltips still have the right types for elements, so I'm not sure if it's necessary?

I can put it back and figure out a solution if it needs to be there.